ci-automation/tapfile_helper_lib.sh

```diff
@@ -128,18 +134,32 @@ function tap_ingest_tapfile() {
             fi
         fi
 
+        local test_output="/dev/null"
+        if [ "${has_error_message}" = "true" ] ; then
+            test_output="${error_message_file}"
+        fi
         SQL="${SQL}INSERT OR IGNORE INTO test_case(name) VALUES ('${test_name}');"
         SQL="${SQL}INSERT OR IGNORE INTO vendor(name) VALUES ('${vendor}');"
 
         SQL="${SQL}INSERT OR REPLACE INTO test_run(run,result,output,case_id,vendor_id)
-                             VALUES ('${run}','${result}', '${error_message}',
+                             VALUES ('${run}','${result}', readfile('${test_output}'),
                                      (SELECT id FROM test_case WHERE name='${test_name}'),
                                      (SELECT id FROM vendor WHERE name='${vendor}'));"
-        error_message=""
+
+        if [ "${has_error_message}" = "true" ] ; then
+            SQL="${SQL}COMMIT;"
```

Comment:
Nitpick: lines like `STUFF="${STUFF}${MORESTUFF}"` could be written like `STUFF+="${MORESTUFF}"` too.

Reply:
Making a mental note!